dashboard/agent.py

```diff
@@ -152,7 +157,42 @@ async def _check_parent():
                 while True:
                     parent = curr_proc.parent()
                     if parent is None or parent.pid == 1 or self.ppid != parent.pid:
-                        logger.error("Raylet is dead, exiting.")
+                        log_path = "/tmp/ray/session_latest/logs/raylet.out"
```

Comment:
I think we should not read session_latest. Maybe passing the session dir properly here?

Reply:
Sounds good.